proxy_wasm_api.h

```diff
@@ -335,6 +335,7 @@ class RootContext : public ContextBase {
                       uint32_t timeout_milliseconds, HttpCallCallback callback);
   // NB: the message is the response if status == OK and an error message
   // otherwise. Returns false on setup error.
+#ifdef PROXY_WASM_PROTOBUF
   WasmResult grpcSimpleCall(StringView service, StringView service_name, StringView method_name,
```

Comment:
What about other gRPC functions? i.e. `open_stream`, `send_message`, `cancel`, and `close`? Shouldn't we be disabling them as well?

Reply:
We could.  I was hoping to talk to others who might be interested in using say flatbuffers with gRPC.  They might have alternative solutions for the functions which depend directly on MessageLite.

Reply:
I'd err on the of side having good support for features that we support, and not having support for features that we don't officially support, i.e. I'm 100% for adding back other functions iff someone adds support for gRPC with flatbuffers, but since we don't have it yet, I think we should disable reminding gRPC functions when compiled without protobuf.

Reply:
There is a bug preventing me from fixing this: https://github.com/envoyproxy/envoy-wasm/issues/508

Reply:
gRPC doesn't have to be a protobuf payload, any buffer (string/string_view) should work and Envoy has support of that. But we should turn off those method with protobuf in its signature.

Reply:
I turned the protobuf ones off, but I didn't provide the "string/string_vew" alternative.   WDYT @PiotrSikora ?  Should I add those.

Reply:
We should probably add those (always, not only when building without protobuf), but it's fine to do that in a follow-up PR.